public/index.html

```diff
@@ -111,6 +428,9 @@ <h3>Thank You, <br>We will contact you soon!</h3>
     <!-- FOOTER -->
     <footer>
       <p><a href="mailto:m@claphelp.life">m@claphelp.life</a></p>
+      <a href="https://github.com/mjimenezleon13/clapHelp" target="_blank">
+        <img style="margin: 10px;" class="icon" src="../public/images/_github-512.png"/>
```

Comment:
The image is still having problems being displayed. This is because of how you're referencing it. change the `src` attribute so that it doesn't include `../public/`, it should be something like `images/image_name.png`